art/defences/preprocessor/preprocessor.py

```diff
@@ -193,3 +244,54 @@ def estimate_forward(self, x: "tf.Tensor", y: Optional["tf.Tensor"] = None) -> "
         :return: Preprocessed data.
         """
         raise NotImplementedError
+
+    def __call__(self, x: np.ndarray, y: Optional[np.ndarray] = None) -> Tuple[np.ndarray, Optional[np.ndarray]]:
+        """
+        Apply local spatial smoothing to sample `x`.
```

Comment:
Line 250-254: I think the documents are not updated. It is for label smoothing.

Reply:
I agree, fixed with 1d7b82217804353137a7e9c916459f8641c80632